app/service/user_service.py

```diff
@@ -38,3 +38,7 @@ def getUserByName(self, name: str) -> User:
     def existsUser(self, name: Optional[str] = None, user: Optional[Union[User,UserIn]] = None, id: Optional[int] = -1) -> bool:
         return id in self.users.keys() or name in self.users.values() or user in self.users.values()
 
+    def updateUser(self, userIn: UserIn, user_id: int):
```

Comment:
this endpoint is not related with the PR. In the future this type of code should be done in a separated PR.

Reply:
this comment is not a blocker